Tests/SentryTests/Integrations/SentryPerformanceTrackerTests.swift

```diff
@@ -0,0 +1,219 @@
+import XCTest
+
+class SentryPerformanceTrackerTests: XCTestCase {
```

Comment:
`m`: This is a good base for tests, thanks 🙏. What is still missing are async tests. Can we please add some so we make sure the synchronization works fine?